cg/services/order_validation_service/workflows/mip_rna/models/order.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import Discriminator, Tag
+from typing_extensions import Annotated
+
+from cg.services.order_validation_service.models.discriminators import has_internal_id
+from cg.services.order_validation_service.models.existing_case import ExistingCase
+from cg.services.order_validation_service.models.order_with_cases import OrderWithCases
+from cg.services.order_validation_service.workflows.mip_rna.models.case import (
+    MipRnaCase,
+)
+
+NewCase = Annotated[MipRnaCase, Tag("new")]
+OldCase = Annotated[ExistingCase, Tag("existing")]
+
+
+class MipRnaOrder(OrderWithCases):
```

Comment:
You probably want to override the delivery type with only the allowed delivery types for this specific order type

Reply:
@islean Could you please check if this is addressed by the new commits?

Reply:
It looks good @ahdamin